packages/gatsby/src/schema/create-key.js

```diff
@@ -1,3 +1,3 @@
 const regex = new RegExp(`[^a-zA-Z0-9_]`, `g`)
 
-module.exports = key => key.replace(regex, `___`)
+module.exports = key => key.replace(`@`, ``).replace(regex, `___`)
```

Comment:
looks like you didn't change the queries to adjust to the commits? or maybe i didn't update the examples. it replaces characters with a single `_` instead of 3 so `frontmatter_updateRole` etc